### DISCUSSION TITLE
Complex icon tracker

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#1821] [General]
Sometimes the source icons that we use are too complicated, if we trace them exactly it results in a cluttered Arcticon. This discussion is a place for reporting if you believe that certain icons are too complicated.

I'll start with feedback I've got recently about Mull: Too much detail around the eye, especially on with the You version.
![afbeelding](https://github.com/Donnnno/Arcticons/assets/31142286/a058f143-dfc9-4fe0-adbb-14db87eff38c)

**Comment by TotallyAvailable** [comment]
While loving the Icon, I can see it and agree. Going with the eyes from/like Bloons TD 6 might work. Also harder to tell for anyone not actually using the icon given the "slight" scaling difference.

There's also the thing with Google Arts & Culture that just looks so wrong. While it comes from the original icon, "eyes getting caught" feels in conflict with the simplicity.

Also the "overly" stretched 'Hasheasily' vs say 'Atomic'. Might make you manually switch for a more "in place" feeling look. Might just be me though.

Assuming someone isn't seeing any previews here right now, totally been dealing with that one. What happened to at least showing the [There is more stuff here you are missing out on] part ?

**Reply by Donnnno** [reply]
I've added this as an alt for google arts:

![image](https://github.com/Donnnno/Arcticons/assets/31142286/eac7da08-6635-40fe-ab4d-d444790ed898)

**Comment by kitters-kat** [comment]
The [retrowars](https://raw.githubusercontent.com/Donnnno/Arcticons/main/icons/black/retrowars.svg) icon is too complex to be the default icon for the app, in my opinion. I like the option of having the stylized CRT screen icon as an alt icon, though, I just think it gets in the way of the simple lines of the arcticons style as it is. We could keep some of the more complex icons as alt icons and have the default be more simplified, so the user has the choice for their own tastes.

**Comment by TotallyAvailable** [comment]
@dgcampbe What about 'Galaxyinvaders' then. Is it a general "complexity" thing or just about the difference to the "min/max possible simplification while still getting the point across" of the original icon.

**Comment by quantenzitrone** [comment]
hamradiotools has some design errors

**Comment by TotallyAvailable** [comment]
I'll follow up on a couple thoughts
(given my lack of actual icon contributions, my point of view might change of course)

I remembered your comment on reworking several Icons with region specific "extras".
I (without any further thought) covered Skype Insiders with the normal Skype Icon, just like Webview **Dev** Tools with the existing Stable Version one.
While covering some stuff that way (duplicates unlikely) works, do people actually care enough to not like a missing "version tag" in the Icons or is it a trade-off that's worth it in terms of simplification -> drop it, if it is obvious/Include potential _alt ver.

As for the previously mentioned Arts & Culture
That's obviously the App Icon...what are the guidelines on how to deal with the "non ideal" decisions a Dev/Company might make.
(Ignore this one if that's just me looking at it wrong)
